examples/all-clusters-app/esp32/main/include/ShellCommands.h

```diff
@@ -134,7 +134,7 @@ class CASECommands
 
 private:
     CASECommands() {}
-    static void OnConnected(void * context, OperationalDeviceProxy * deviceProxy)
+    static void OnConnected(void * context, Messaging::ExchangeManager & exchangeMgr, SessionHandle & sessionHandle)
```

Comment:
Passing in the exchange manager here is pretty weird.  I'd expect that anyone who can get their hangs on a CASESessionManager to make the call that lands in here should also be able to get to an ExchangeManager....

I understand the "this is expedient, and keeps code churn down" argument, but there should be a TODO and a followup to stop doing this and just hand back the session.